src/TrackPanel.cpp

```diff
@@ -810,6 +834,30 @@ void TrackPanel::OnContextMenu(/*AudacityProject &project, */ wxCommandEvent& ev
     }
 }
 
+void TrackPanel::OnTrackAreaContextMenu(wxCommandEvent& event)
+{
+    switch(event.GetId())
+    {
```

Comment:
And I don't like this switch statement.  Instead you should be able to get an ID from `event`, index into the table that describes the menu, and invoke CommandManager::HandleTextualCommand.